src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -33,34 +40,73 @@ public AddressBook(ReadOnlyAddressBook toBeCopied) {
         persons.setPersons(toBeCopied.getPersonList());
     }
 
-    //// list overwrite operations
+    //// undo & redo operations
 
     /**
-     * Makes a copy of persons, and stores it in personsList.
+     * Empties the redoList.
      */
-    private void save() {
+    private void clearRedoList() {
+        while (!redoStack.empty()) {
+            redoStack.pop();
+        }
```

Comment:
This method body can be replaced with `redoStack.clear()`.

Reply:
Nice find! It was absent in the javadoc website so I didn't know it existed.